Source/WebKit/UIProcess/Cocoa/_WKWarningView.mm

```diff
@@ -159,8 +161,7 @@
 #else
     UIColor *red = [UIColor colorWithRed:0.998 green:0.239 blue:0.233 alpha:1.0];
     UIColor *darkGray = [UIColor colorWithRed:0.118 green:0.118 blue:0.118 alpha:1.0];
-    UIColor *lighterGray = [UIColor colorWithRed:0.220 green:0.224 blue:0.231 alpha:1.0];
-    UIColor *mediumGray = [UIColor colorWithRed:0.729 green:0.733 blue:0.741 alpha:1.0];
+    UIColor *lighterGray = [UIColor colorWithRed:0.937 green:0.937 blue:0.937 alpha:1.0];
     UIColor *white = [UIColor whiteColor];
```

Comment:
Seeing hardcoded colors is always concerning 😅

Does this look okay in dark mode? And high contrast mode?

Note that `UIColor` has `+darkGrayColor`, `+lightGrayColor` and `+grayColor`.

Reply:
Yeah... `+lightGrayColor` is still too dark (70% grayscale, whereas I need something like ~90%). `+systemColor6Color` is basically what I want in light mode, except now I need to adjust the text font color, as well, so it adapts to dark mode. Currently, these warning pages in Safari don't adapt to appearance changes on iOS at all. I'd almost rather not adapt so this View remains consistent with the other warning pages.

Reply:
Makes sense, dark mode sounds like a possible future enhancement.